connectors/src/pages/_app.tsx

```diff
@@ -0,0 +1,9 @@
+import { type AppType } from "next/dist/shared/lib/utils";
```

Comment:
We might need it in the future, so should we keep it as an example of how to implement it or just remove it and figure out later?

Reply:
This is a regular next app right?

let’s remove as much as we can for now 👍